.github/workflows/tests.yml

```diff
@@ -96,9 +96,12 @@ jobs:
       run: pytest -v tests/
 
   binja-tests:
-    name: Binary Ninja tests for ${{ matrix.python-version }} on ${{ matrix.os }}
+    name: Binary Ninja tests for ${{ matrix.python-version }}
     runs-on: ubuntu-20.04
     needs: [code_style, rule_linter]
+    # do not run on forks, because:
+    # with the exception of GITHUB_TOKEN, secrets are not passed to the runner when a workflow is triggered from a forked repository.
+    if: "! github.event.pull_request.head.repo.fork"
```

Comment:
very cool!

Reply:
I'm still not sure this works as expected :/

Reply:
It worked in https://github.com/mandiant/capa/actions/runs/4788977898/jobs/8516324797